src/API/Site/Controllers/MerchantCenter/PhoneVerificationController.php

```diff
@@ -0,0 +1,165 @@
+					'callback'            => $this->get_request_phone_verification_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+					'args'                => [
+						'phone_region_code' => [
+							'description'       => __( 'Two-letter country code (ISO 3166-1 alpha-2) for the phone number.', 'google-listings-and-ads' ),
+							'required'          => true,
+							'type'              => 'string',
+							'validate_callback' => 'rest_validate_request_arg',
+						],
+						'phone_number'        => [
+							'description'       => __( 'The phone number to verify.', 'google-listings-and-ads' ),
+							'required'          => true,
+							'type'              => [ 'integer', 'string' ],
+							'validate_callback' => 'rest_validate_request_arg',
+						],
+						'verification_method' => $verification_method,
+					],
+				]
+			]
+		);
+
+		$this->register_route(
+			'/mc/phone-verification/verify',
+			[
+				[
+					'methods'              => TransportMethods::CREATABLE,
+					'callback'            => $this->get_verify_phone_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+					'args'                => [
+						'verification_id'           => [
+							'description'       => __( 'The verification ID returned by the /request call.', 'google-listings-and-ads' ),
+							'required'          => true,
+							'type'              => 'string',
+							'validate_callback' => 'rest_validate_request_arg',
+						],
+						'verification_code' => [
+							'description'       => __( 'The verification code that was sent to the phone number for validation.', 'google-listings-and-ads' ),
+							'required'          => true,
+							'type'              => 'string',
+							'validate_callback' => 'rest_validate_request_arg',
+						],
+						'verification_method' => $verification_method,
+					],
+				]
+			]
+		);
+	}
+
+	/**
+	 * Get callback for requesting phone verification endpoint.
+	 *
+	 * @return callable
+	 */
+	protected function get_request_phone_verification_callback(): callable {
+		return function( Request $request ) {
+			try {
+				$verification_id = $this->phone_verification->request_phone_verification(
+					$request->get_param( 'phone_region_code' ),
+					new PhoneNumber( (string) $request->get_param( 'phone_number' ) ),
+					$request->get_param( 'verification_method' ),
+				);
+				return [
+					'verification_id' => $verification_id
+				];
+			}
+			catch ( Exception $e ) {
+				return $this->response_from_exception( $e );
+			}
+		};
+	}
+
+	/**
+	 * Get callback for verifying a phone number.
+	 *
+	 * @return callable
+	 */
+	protected function get_verify_phone_callback(): callable {
+		return function( Request $request ) {
+			try {
+				$this->phone_verification->verify_phone_number(
+					$request->get_param( 'verification_id' ),
+					$request->get_param( 'verification_code' ),
+					$request->get_param( 'verification_method' ),
+				);
+			}
```

Comment:
I guess just returns an empty `200`, is that right? Should we include some sort of `status`, or is that overkill?

Reply:
We can return a 204 status code even 🚶‍♂️

Reply:
Update in e06473232a3d2c27edeae672b7dc08e4b31bf73a